source/main/cardano/CardanoNode.js

```diff
@@ -293,7 +293,7 @@ export class CardanoNode {
           nodeImplementation,
           cliPath,
           // TODO: Make this dynamic
-          nodePort: 8888,
+          nodePort: 3100,
```

Comment:
this change may not be needed, now that the other file specifies `--rest-listen`